<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DragWidget</class>
 <widget class="QWidget" name="DragWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DragWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbObjectsetting">
     <property name="title">
      <string>object setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbtListSource">
        <property name="text">
         <string>listSource</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtListWidget">
        <property name="text">
         <string>listWidget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtTreeWidget">
        <property name="text">
         <string>treeWidget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtTableWidget">
        <property name="text">
         <string>tableWidget</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QGroupBox" name="gbParametersetting">
     <property name="title">
      <string>parameter setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_1">
        <property name="text">
         <string>acceptDrags</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>dragEnabled</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>dragDropMode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>defaultDropAction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbListSource">
     <property name="title">
      <string>listSource</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listSource">
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gbListWidget">
     <property name="title">
      <string>listWidget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="gbTreeWidget">
     <property name="title">
      <string>treeWidget</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="gbTableWidget">
     <property name="title">
      <string>tableWidget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>b</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>c</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
